server/src/main/java/org/elasticsearch/client/support/AbstractClient.java

```diff
@@ -1185,6 +1188,36 @@ public DeleteStoredScriptRequestBuilder prepareDeleteStoredScript(){
         public DeleteStoredScriptRequestBuilder prepareDeleteStoredScript(String id){
             return prepareDeleteStoredScript().setId(id);
         }
+
```

Comment:
Same story as for the action names. We should consider whether to include this under indices or under a new "client" interface? Will add to our weekly sync, this should not block merging this.